library/core/src/ptr/mut_ptr.rs

```diff
@@ -2361,6 +2361,78 @@ impl<T: ?Sized> PartialOrd for *mut T {
 #[unstable(feature = "kani", issue = "none")]
 mod verify {
     use crate::kani;
+    use core::mem;
+    // Constant for array size used in all tests
+    const ARRAY_SIZE: usize = 5;
```

Comment:
Could you please add a further comment saying _why_ you have chosen this particular value?

Reply:
Comment added.